src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1774,59 +1774,45 @@ public void sendPluginMessage(Plugin source, String channel, byte[] message) {
 
     private void pulseTickMap() {
```

Comment:
Also, that would break with block that need to change their tick speed without resetting pulse; like command blocks for instance. I would just keep the current system.